recipes/freetype/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.5.5" %}
+
+package:
+    name: freetype
+    version: {{ version }}
+
+source:
+    fn: freetype-{{ version }}.tar.bz2
+    url: http://download.savannah.gnu.org/releases/freetype/freetype-{{ version }}.tar.bz2
+    sha256: 387bcc4b780b12484aa2ec9f7db1a55d8286eb5639f45fbc0fbba7a4e5a1afb9
+    patches:
+        - subpixel.patch
```

Comment:
Normally, people don't write a page about how they shouldn't be infringing on any patents. They also don't tend add...

> ...you should do that at your own risk.

As they point out in there, the issue is displaying the rendered fonts, which would happen in an application. Still feels pretty fuzzy there.